packages/fxa-settings/src/pages/CompleteResetPassword/index.tsx

```diff
@@ -0,0 +1,188 @@
+            <h1 id="fxa-reset-link-expired-header" className="card-header">
+              Reset password link expired
+            </h1>
+          </FtlMsg>
+
+          <FtlMsg id="complete-reset-pw-link-expired-message">
+            <p className="mt-4">
+              The link you clicked to reset your password is expired.
+            </p>
+          </FtlMsg>
+          <FtlMsg id="complete-reset-pw-resend-link">
+            <button
+              onClick={resendLinkHandler}
+              className="cta-primary cta-base-p mt-4"
+            >
+              Receive new link
+            </button>
+          </FtlMsg>
+        </>
+      )}
+      {isLinkDamaged && (
+        <>
+          <FtlMsg id="complete-reset-pw-link-damaged-header">
+            <h1 id="fxa-reset-link-damaged-header" className="card-header">
+              Reset password link damaged
+            </h1>
+          </FtlMsg>
+
+          <FtlMsg id="complete-reset-pw-link-damaged-message">
+            <p className="mt-4">
+              The link you clicked was missing characters, and may have been
+              broken by your email client. Copy the address carefully, and try
+              again.
+            </p>
+          </FtlMsg>
+        </>
+      )}
+
+      {/* With valid password reset link */}
+      {!isLinkExpired && !isLinkDamaged && (
+        <>
+          <FtlMsg id="complete-reset-pw-header">
+            <h1 id="fxa-reset-link-damaged-header" className="card-header">
+              Create new password
+            </h1>
+          </FtlMsg>
+
+          {showSyncWarning && (
+            <WarningMessage
+              warningMessageFtlId="complete-reset-password-warning-message"
+              warningType="Remember:"
+            >
+              When you reset your password, you reset your account. You may lose
+              some of your personal information (including history, bookmarks,
+              and passwords). That’s because we encrypt your data with your
+              password to protect your privacy. You’ll still keep any
+              subscriptions you may have and Pocket data will not be affected.
+            </WarningMessage>
+          )}
+          {showAccountRecoveryInfo && (
+            <FtlMsg id="complete-reset-pw-account-recovery-info">
+              <p className="mb-5 mt-4">
+                You have successfully restored your account using your account
+                recovery key. Create a new password to secure your data, and
+                store it in a safe location.
+              </p>
+            </FtlMsg>
+          )}
+          {/* Hidden email field is to allow Fx password manager
+           to correctly save the updated password. Without it,
+           the password manager tries to save the old password
+           as the username. */}
+          <input type="email" value={email} className="hidden" />
+          <section className="text-start mt-4">
+            <FormPassword
```

Comment:
Do we want to ditch the PW balloon? It's pretty much Tailwindified, but not sure what we want here in the long run. Since this is functionally different, when we use functional tests here they'll have to be tweaked (all ok by me, just calling it out).

Reply:
Ah yes, I'd used the `FormPassword` component to quickly include the password validation, but it does have some notable differences (mainly, password requirements above fields, no bubble, and different button text) from the original CompleteResetPassword form).

From an accessibility perspective, I think the FormPassword format might be preferable to the bubble, but I'm not sure.

Reply:
@vpomerleau The `FormPassword` is more accessible IMO but the PW balloon also isn't bad. Personally I'd run this by Vesta on which she'd prefer and file an issue if she wants the balloon. We may want this to match parity exactly at first, I'm not sure.